README.md

```diff
@@ -27,6 +27,13 @@ The buildpack will do the following for PHP applications:
 * Contributes a PHP agent to a layer and configures `$PHP_INI_SCAN_DIR` to use it
 * Transforms the contents of the binding secret to environment variables with the pattern `NEW_RELIC_<KEY>=<VALUE>`
 
+The buildpack will do the following for Python applications:
+
+* Copies configuration file in /resources/newrelic.ini to the root folder of your application
```

Comment:
```suggestion
* Installs a default `newrelic.ini` configuration file to the root folder of your application
```

Reply:
Also, what if this file exists already? I think the way it is right now, the buildpack would fail to copy the file and overwrite the user's file, but it would fail the build which isn't great.

You might want to either have the buildpack just backoff if it tries to copy the file and gets an error saying that the file exists. In Go, you typically try the action and check the error type versus checking if the file exists in advance.

You could then also add a note in the README, like "You may override this file by including a newrelic.ini file at the root of your application".

Reply:
Done